refs/refs-internal.h

```diff
@@ -650,6 +650,9 @@ typedef int read_raw_ref_fn(struct ref_store *ref_store, const char *refname,
 typedef int read_symbolic_ref_fn(struct ref_store *ref_store, const char *refname,
 				 struct strbuf *referent);
 
+typedef int fsck_fn(struct ref_store *ref_store,
+		    struct fsck_refs_options *o);
```

Comment:
Don't we need a forward declaration of this struct?

Reply:
I have a forward declaration in `refs.h`. I guess I need to still do a forward declaration here to avoid misunderstanding.